src/test/test-thp-disable-completely.c

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#include "test-thp-util.h"
+
+static int intro(void) {
+        int r = prctl(PR_GET_THP_DISABLE, 0, 0, 0, 0);
```

Comment:
Rather than introduce three executable, how about to print the result of the prctl() in a string?